query-engine/connector-test-kit-rs/query-engine-tests/tests/new/update_no_select.rs

```diff
@@ -0,0 +1,28 @@
+use query_engine_tests::*;
+
+#[test_suite]
+mod update_with_no_select {
+    pub fn occ_simple() -> String {
+        include_str!("occ_simple.prisma").to_owned()
+    }
+
+    #[connector_test(schema(occ_simple))]
+    async fn update_with_no_select(mut runner: Runner) -> TestResult<()> {
+        let create_one_resource = r#"
+        mutation {
+            createOneResource(data: {id: 1}) {
+              id
+            }
+          }"#;
+
+        runner.query(create_one_resource).await.unwrap().to_json_value();
+
+        let logs = runner.get_logs().await;
```

Comment:
Started peeking the review from this line, which shows how the intended API is used.